src/components/reducers/GlobalFilter.tsx

```diff
@@ -30,12 +30,7 @@ export default function GlobalFilter(globalFilterProps: GlobalFilterProps) {
           onChange(e.target.value);
         }}
         placeholder={`${count} records...`}
-        style={{
```

Comment:
About this erase of style. It is necessary to lose the background property?

Reply:
I removed all the styling (including the background property) since it'd be handled by the styling of `input[type=search]` in the user's theme.